e2e/e2etests/test_eth_deposit.go

```diff
@@ -20,16 +20,29 @@ import (
 )
 
 // TestEtherDeposit tests deposit of ethers
-func TestEtherDeposit(r *runner.E2ERunner) {
-	hash := r.DepositEtherWithAmount(false, big.NewInt(10000000000000000)) // in wei (0.01 eth)
+func TestEtherDeposit(r *runner.E2ERunner, args []string) {
+	defaultAmount := big.NewInt(10000000000000000) // default value of 0.01 ETH in wei
```

Comment:
Maybe we can group all the default argument as part of the `E2ETest` object in `e2etests.go`?

Reply:
We can add the argument as we test the different commands on live network. Though we can already do it for the basic tests:
- eth withdraw
- eth deposit
- erc20 withdraw
- erc20 deposit
- btc withdraw
- btc deposit